client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/ClinicalDetails.tsx

```diff
@@ -91,24 +44,24 @@ const ClinicalDetails: React.FC = (): JSX.Element => {
         context.setClinicalDetailsData({...context.clinicalDetailsData as ClinicalDetailsData, [fieldToUpdate]: updatedValue});
     };
 
-    const handleSymptomCheck = (symptom: Check) => {
-        if (selectedSymptoms.find(checkedSymptom => checkedSymptom === symptom.name)) {
-            setSelectedSymptoms(selectedSymptoms.filter((checkedSymptom) => checkedSymptom !== symptom.name));
+    const handleSymptomCheck = (symptom: string) => {
+        if (selectedSymptoms.find(checkedSymptom => checkedSymptom === symptom)) {
```

Comment:
You can use contains here since its a string array

Reply:
contains זה לstring אבל שיניתי לincludes